ipam/allocator.go

```diff
@@ -787,15 +815,21 @@ func (alloc *Allocator) update(sender mesh.PeerName, msg []byte) error {
 		alloc.nicknames[peer] = nickname
 	}
 
+	var oldRanges []address.Range
 	switch {
 	// If someone sent us a ring, merge it into ours. Note this will move us
 	// out of the awaiting-consensus state if we didn't have a ring already.
 	case data.Ring != nil:
+		oldRanges = alloc.ring.OwnedAndMergedRanges()
 		updated, err := alloc.ring.Merge(*data.Ring)
 		switch err {
 		case nil:
 			if updated {
 				alloc.pruneNicknames()
+				err := alloc.monitor.HandleUpdate(oldRanges, alloc.ring.OwnedAndMergedRanges())
+				if err != nil {
+					return err
+				}
```

Comment:
At all other `HandleUpdate` call sites an error is logged but ignored. But not here. Why?